src/brogue/Recordings.c

```diff
@@ -28,6 +28,11 @@
 
 #define RECORDING_HEADER_LENGTH     32  // bytes at the start of the recording file to store global data
 
+static const int keystrokeTable[] = {UP_ARROW, LEFT_ARROW, DOWN_ARROW, RIGHT_ARROW,
+    ESCAPE_KEY, RETURN_KEY, DELETE_KEY, TAB_KEY, NUMPAD_0, NUMPAD_1,
+    NUMPAD_2, NUMPAD_3, NUMPAD_4, NUMPAD_5, NUMPAD_6, NUMPAD_7, NUMPAD_8, NUMPAD_9};
+
+
```

Comment:
brogue is inconsistent, I'm assuming 2 blank lines intentionally

Reply:
Good catch, I'll try to be consistent with that, but personally i'm not fussed about an extra line between things, I care mostly about formatting of control structures etc. for readability purposes